### DISCUSSION TITLE
Random Lag

### DISCUSSION THREAD

**Discussion by GiveMeFox** [ppy/osu#22904] [Q&A]
os: 6.2.2-arch2-1
osu: 2023.305.0
I've been getting random lag while playing

**Comment by Axilotl17** [comment]
I use arch and its fine on all of my computers. It's your setup.

We also need log files to do anything at all. If you really believe its an issue then upload those.

If you use NVIDIA drivers, specifically `nvidia-settings`, then try turning on "Force Composition Pipeline"; it fixes a bunch of openGL rendering issues.
